integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/CarbonScanRDD.scala

```diff
@@ -232,6 +237,10 @@ class CarbonScanRDD(
           val value = reader.getCurrentValue
           value
         }
+        private def close() {
+          val carbonSplit = split.asInstanceOf[CarbonSparkPartition]
+          inputMetricsStats.updateBytesRead(carbonSplit.split)
```

Comment:
It is a wrong statistic we are providing to spark, it supposed to be how much we read from carbon file. This logic is current only if each block has only one blocklet.